src/main/groovy/org/mockito/release/internal/gradle/VersioningPlugin.java

```diff
@@ -62,4 +68,14 @@ public void execute(BumpVersionFileTask t) {
             }
         });
     }
+
+    private void createVersionPropertiesFile(Project project, File versionFile) {
+        LOG.lifecycle("  Required file version.properties doesn't exist. Creating it automatically. Remember about checking it into VCS!");
+        LOG.lifecycle("  Initial project version in version.properties set to {}", project.getVersion());
+        String versionFileContent = "#Version of the produced binaries. This file is intended to be checked-in.\n"
+                + "#It will be automatically bumped by release automation.\n"
+                + "version=" + project.getVersion() + "\n";
```

Comment:
Great catch on the problem of 'unspecified' version! Let's handle this situation cleanly. I'll suggest and approach in the comment to the PR.